### DISCUSSION TITLE
【要望】質問の回答へ返信したものも、マイページの「回答した質問」に表示されるようにしてほしい

### DISCUSSION THREAD

**Discussion by Yotty0404** [increments/qiita-discussions#615] [要望・アイデア・不具合]
質問の回答ではなく、回答への返信をしたものに関しては、現状マイページの「回答した質問」に表示されないようでした。
最近質問にコメントしたもので、どういった内容を書いたかなーと探そうと思ったときに「回答した質問」に表示されておらず、探すのが少し不便に感じました。

返信したものも「回答した質問」に表示されているとすごいありがたいです。

立て続けに申し訳ないですが、ぜひご検討いただけますと幸いです

**Comment by tanaka-asami415** [comment]
@Yotty0404 
リクエストありがとうございます🙏

確かに回答コメントも表示されている方がいいですよね。
パフォーマンス周りの影響もあるため、確認しながら対応を検討してみます！
